docs/docs/03-language-reference.md

```diff
@@ -74,6 +74,15 @@ import TOCInline from '@theme/TOCInline';
 > let q: num? = nil;          // q is an optional num
 > ```
 
+Numeric literals can be formatted and padded with extra zeroes or underscores to make them easier to read.
+These don't affect the value of the number:
```

Comment:
I see the use cases, but I wonder if it will lead to confusion. I.E.
```js
let price = 00.50;
log("Your total is: ${price}");
```

Will users expect this padding to substitute numerical formatting?  

Im for sure game to try, but maybe we need to be clear this is not a formatting feature.

Reply:
I'll update the docs